.github/workflows/ci.yml

```diff
@@ -111,15 +111,15 @@ jobs:
 
       - name: Run tests
         run: |
-          if [ ${{ runner.os }} == "Linux" ]; then
-            sudo docker run --rm -d -e POSTGRES_PASSWORD=test123 -p 5432:5432 postgres:15.4-alpine3.18
-          fi
-
           postgres_enabled=0
           if [ ${{ runner.os }} == "Linux" ]; then
+            sudo docker run --rm -d -e POSTGRES_PASSWORD=test123 -p 5432:5432 postgres:15.4-alpine3.18
             postgres_enabled=1
           fi
 
+          export MAKEFLAGS="-j1"
+          export NIMFLAGS="--colors:off -d:chronicles_colors:none"
+          
```

Comment:
That LGTM but shall we add that in a separate PR so that is clear the commit that applied it?

Reply:
> That LGTM but shall we add that in a separate PR so that is clear the commit that applied it?

@NagyZoltanPeter - maybe easier to just update the PR title and description to also reflect the CI test change :)

Reply:
:-) agree! I do it.